src/lib/constants.js

```diff
@@ -4,4 +4,59 @@ export const HEADERS = {
   JSON: 'application/json',
   JSON_CHARSET: 'application/json; charset=utf-8',
   OCTET_STREAM: 'application/octet-stream',
+  GZIP: 'gzip',
 };
+
+export const HEADER_TYPE = {
```

Comment:
I like this approach. Easy to maintain constants. 👍

Reply:
We need to create a `@verdaccio/error-utils` or something like that to share error messages with plugins, right now is kind of wild west where the plugin dev can set whatever the wants